Documentation/docs_site_update.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# Simple script to check if build should run all lint. Will return empty result if no need or a string explaining why yes.
+
+if [ -n "$NIGHTLY" ]; then
+    echo "NIGHTLY env var is set: $NIGHTLY. Skipping Content Docs Update"
+    exit 0
+fi
+
+if [ -n "$INSTANCE_TESTS" ]; then
+    echo "INSTANCE_TESTS env var is set: $INSTANCE_TESTS. Skipping Content Docs Update"
+    exit 0
+fi
+
+if [ -z "$CIRCLE_BRANCH" ]; then
+    # simply compare against origin/master. Local testing case..
+    DIFF_COMPARE=origin/master  # disable-secrets-detection
+elif [ "$CIRCLE_BRANCH" == "master" ]; then
+    # on master we use the range obtained from CIRCLE_COMPARE_URL
+    # example of comapre url: https://github.com/demisto/content/compare/62f0bd03be73...1451bf0f3c2a
+    # if CIRCLE_COMPARE_URL is not set we use last commit
+    if [ -z "$CIRCLE_COMPARE_URL" ]; then
+        DIFF_COMPARE="HEAD^1...HEAD"
+    else
+        DIFF_COMPARE=$(echo "$CIRCLE_COMPARE_URL" | sed 's:^.*/compare/::g')    
```

Comment:
why not always use `HEAD^1...HEAD`?

Reply:
It depends on how the merge was done and the build triggered. The build can have multiple commits so HEAD^1 is not accurate. So compare url is preferred. There are also some edge scenarios that circle compare url is empty.